### PR TITLE
Use ansible_python_interpreter=auto

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request makes the change of using `ansible_python_interpreter=auto` wherever possible.  I also went ahead and pulled in the latest upstream changes from [cisagov/skeleton-packer](https://github.com/cisagov/skeleton-packer).

## 💭 Motivation and Context

In the future, `ansible_python_interpreter=auto` will be Ansible's default behavior, so it makes sense to go ahead and prepare for it.  Since we remove python2, it should not change the AMI that is generated.

## 🧪 Testing

I have successfully generated a staging AMI with these changes.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Defend today, secure tomorrow! 🦅

**Review by mcdonnnj — APPROVED** [review]
The times they are a-changin' 🎶

File: terraform-post-packer/backend.tf [hunk 1]
```diff
@@ -5,6 +5,6 @@ terraform {
     dynamodb_table = "terraform-state-lock"
     profile        = "cool-terraform-backend"
     region         = "us-east-1"
-    key            = "freeipa-server-packer/terraform-test-user.tfstate"
+    key            = "skeleton-packer/terraform-post-packer.tfstate"
```
Comment (dav3r) [inline terraform-post-packer/backend.tf]: This key should be: `freeipa-server-packer/terraform-post-packer.tfstate`

**Review by felddy — APPROVED** [review]
Looks good.  See @dav3r 's mod.

**Review by dav3r — APPROVED** [review]
👍 with one small, but important change requested.

File: terraform-post-packer/backend.tf (hunk 1 above)
Comment (jsf9k) [reply terraform-post-packer/backend.tf]: Please have a look at commit 8ca1ace and let me know if I caught everything.

Comment (dav3r) [reply terraform-post-packer/backend.tf]: Yes, that all looks correct to me- good catch on the providers and caller identity!

**Pull request merged**